src/pages/en/guides/publish-to-npm.md

```diff
@@ -111,21 +114,21 @@ The entry points allowed by Astro to import your component or any of its [files]
 }
 ```
 
-In this example, importing `my-component` would use `index.js`, while importing `my-component/astro` or `my-component/react` would use `MyAstroComponent.astro` or `MyReactComponent.jsx`.
+In this example, importing `my-component` would use `index.js`, while importing `my-component/astro` or `my-component/react` would use `MyAstroComponent.astro` or `MyReactComponent.jsx` respectively.
 
-#### `package.json#files`
+#### `files`
 
 ```json
 {
   "files": ["index.js", "MyAstroComponent.astro", "MyReactComponent.jsx"]
 }
 ```
 
```

Comment:
What do folks think about removing this? `files` is an optional optimisation to exclude unnecessary files from the bundle shipped via npm, but it’s also a footgun: if a beginner gets all this set up and then later adds a new file without updating this, they might be confused as to why their new file is missing from the npm package.

```suggestion
```

Reply:
Interesting. This is outside my area of expertise, but I get the problem and I don't like setting people up for failure.

If this is a common/forseable problem, then what about a statement basically saying just that? Something like:

"This is an optional optimization to exclude unnecessary files from the bundle shipped to users via npm. Note that **only files listed here will be included in your package**, so if you add or change files necessary for your package to work, you must update this list accordingly."

???

Reply:
Sure! That reads perfectly.

I’ve definitely released my share of patch versions where the changelog is “Actually include new file” because of `files`, so based on a sample of one, this is common problem 😂

Reply:
Added! Thanks!